### ISSUE TITLE
Emission layer always exports as 1.0 even if painted with other values

### ISSUE THREAD

**Issue by Hoek67** [armory3d/armortools#968]
**ArmorPaint version:**
v0.8 4/6/2021 (grabbed latest)

**OS/device including version:**
Windows 10 x64 NVidia 1050

**Issue description:**
* Mode has no checkbox for emission (ie. other attributes do - roughness, opacity, base color, etc) which is a bit odd given emission is an attribute of the material output.
* Also emission seems to be always exporting a 1.0 regardless of setting.

**Steps to reproduce:**
Use a brush and paint with emission 0.0 0.25 0.5 etc
Export "emis" layer and basically it's all pure white (1.0). From what I can tell it should be like roughness,  metallic, etc  (grayscale)

I use 3DSMAX and basically the emission/self illumination makes certain parts of the texture "glow".

I have spent hours trying different things to make sure not an error on my behalf. From what I can tell, the ability to export "emis" was done not too long ago.

**Comment by Bjarrrne** [comment]
Yes. Exact same problem here (ArmorPaint.org - v0.8 (2021-09-06) - e11cf49). "emis" layer always exports as white image.

**Comment by luboslenco** [comment]
Emission texture should at least export now in binary form (black for 0.0, white for 1.0). Also `Mode - Emission` is now present. Will do more emission fixes at https://github.com/armory3d/armortools/issues/1298. Thanks for report.
